### PR TITLE
*: update contributing guide

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

## What have you changed? (mandatory)

Add some description of handling CI test.

## What are the type of the changes? (mandatory)
- Misc (other changes)

## How has this PR been tested? (mandatory)
No.

## Does this PR affect documentation (docs) update? (mandatory)
No.

## Does this PR affect tidb-ansible update? (mandatory)
No.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
...
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
+
+### Handling CI
+
+We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a
+PR for the first time, you will need someone of us to comment`/ok-to-test` to
+trigger the CI. After this, new changes on the branch will be able to trigger
+the test automatically. You may also manually run the test by commenting
+`/test`.
```
Comment (breeswish) [inline CONTRIBUTING.md]: @zhouqiang-cl Is `/rebuild` and `/test` the same?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
...
 The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain a `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)
 
-You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
+
+### Handling CI
+
+We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a
+PR for the first time, you will need someone of us to comment`/ok-to-test` to
```
Comment (Connor1996) [inline CONTRIBUTING.md]:
```suggestion
PR for the first time, you will need someone of us to comment `/ok-to-test` to
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
...
+trigger the CI. After this, new changes on the branch will be able to trigger
+the test automatically. You may also manually run the test by commenting
```
Comment (Connor1996) [inline CONTRIBUTING.md]:
```suggestion
the test automatically. You can also manually run the test by commenting
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (disksing) [reply CONTRIBUTING.md]: Tried `/rebuild`, seems to be the same.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
...
-You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
+
+### Handling CI
+
+We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a
+PR for the first time, you will need someone of us to comment `/ok-to-test` to
+trigger the CI. After this, new changes on the branch will be able to trigger
+the test automatically. You can also manually run the test by commenting
```
Comment (Hoverbear) [inline CONTRIBUTING.md]:
```suggestion
the test automatically. You will also be able manually run the test by commenting
```

Comment (disksing) [reply CONTRIBUTING.md]: Signed-off-by: disksing <i@disksing.com>

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
...
 The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain a `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)
 
-You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
+
+### Handling CI
+
+We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a
+PR for the first time, you will need someone of us to comment `/ok-to-test` to
```
Comment (siddontang) [inline CONTRIBUTING.md]: seem the first commit can still trigger the CI, right?

Comment (disksing) [reply CONTRIBUTING.md]: No, for security reasons, only users in the whitelist can trigger the CI.

**Comment by disksing** [comment]
Only document changes, low risk, so I'm going skip CI and merge it.

**Pull request merged**